### PR TITLE
cat testresults.xml on travis

### PR DESCRIPTION
**Pull request by domsj**
This will bring more clarity for what I actually wanted to know in https://github.com/openvstorage/alba/pull/205

### CODE REVIEW

File: travis.sh [hunk 1]
```diff
@@ -179,8 +179,10 @@ script () {
             true
             ;;
         system2)
-            ${TEST_DRIVER} ocaml | tail -n256
-            expr $PIPESTATUS && false
+            ${TEST_DRIVER} ocaml | tail -n1000
+            X=$PIPESTATUS
```
Comment (toolslive) [inline travis.sh]: $PIPESTATUS[0] ?

**Comment by domsj** [comment]
Ok, this change exposes the culprit for the sporadic test failures:
```

	<testcase classname="all.alba_test" name="test_create_namespaces" status="run" time="57.081463">
		<error message="Alba_client_errors.Error.Exn(6)"></error>
	</testcase>
```

Will have a deeper look at what's going wrong with this one at a later time...

**Comment by toolslive** [comment]
ok.

File: travis.sh (hunk 1 above)
Comment (domsj) [inline travis.sh]: It seems to work with just $PIPESTATUS, so I think in this case they are equivalent.

**Pull request merged**